src/charm.py

```diff
@@ -53,29 +52,27 @@
     service_running,
     service_stop,
 )
-from charms.prometheus_k8s.v0.prometheus_scrape import (
-    MetricsEndpointAggregator,
-    _type_convert_stored,
-)
+from charms.prometheus_k8s.v0.prometheus_scrape import _type_convert_stored
```

Comment:
Separate issue: move `_type_convert_stored` to cos-lib? It's duplicated all over our charms.
Here specifically seems silly to `fetch-lib` this charmlib just for this helper, if we already depend on cos-lib.